### ISSUE TITLE
connection

### ISSUE THREAD

**Issue by sinku252** [openshopio/openshop.io-android#45]
how to connect app to my woocommerce website

**Comment by abuzer** [comment]
looking for same question.

**Comment by basitanwer** [comment]
You'll have to create a web server for this mobile application.

**Comment by rahulsinghmzp** [comment]
@basitanwer i have woocommerce based website but unable to connect wit tha app as I am unable to get Organization_id required in the app. How to get Organization_id required in Endpoint and Const java file of the app. Please help.
Thanks.

**Comment by basitanwer** [comment]
If you take a look at the [const file](https://github.com/openshopio/openshop.io-android/blob/66f0e7cdc0ca9c7b26ebd169b5a97fd03b71e87c/app/src/main/java/bf/io/openshop/CONST.java) you'll see the organization id is 4.

The best way to connect this app to your server is to install wireshark and listen to all the http requests (the json request and recieved). Then implement the json consumption accordingly.

Tell me if anything is unclear.

**Comment by rahulsinghmzp** [comment]
@basitanwer Yes organization_id is 4 but does it will work for my website also? Or it will be generated for my website when integrated with apiary.io.
I am getting the json response from my website at url====  https://www.zerggo.com/wp-json/wc/v2 .
i placed this url in the endpoint java file for API_URL but getting internal error due to organization_id.

Also please tell what are the end points which i should use so that it app can communicate with web data. my website is  https://www.zerggo.com.
 I have enabled REST API on my woocommerce website.
where to put the customer and secret ket in the app for app and website integration?

i have done facebook_app_id and google gcm integration.

please help. Thanks

**Comment by basitanwer** [comment]
These are just to many questions to answer.

Look, openshop.io is an opensource mobile application but that doesn't mean they want you to use it to go on creating your own application (i think). They want you to contact them to help you create a web server for a cost. 

Their server goes down constantly, their documentation for the web server API is outdated and they are not actively participating in the issue tracker.

How i worked around was i snooped the requests sent and received using wireshark and adjusted the web server or at some point the app accordingly. If you are willing to do these hacks then please listen to the JSON data to and fro.

**Comment by nabil6391** [comment]
bro @basitanwer if you could post the json datas here that would be great

**Comment by basitanwer** [comment]
Thats a lot of data. Can't you extract the data?

**Comment by nabil6391** [comment]
Since the server is not running anymore i am unable to it... Please see if
you can provide even a subset of the json data...

I just need this to run a dummy server and to understand how each object is
mapped
